perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/AttributesManagerBlImpl.java

```diff
@@ -5594,6 +5597,9 @@ private Attribute mergeAttributeValue(PerunSession sess, Attribute attribute, Pe
 		} else //noinspection ConstantConditions
 			if (primaryHolder instanceof Member) {
 				getPerunBl().getAttributesManagerBl().setAttribute(sess, (Member) primaryHolder, attribute);
+			} else //noinspection ConstantConditions
+				if (primaryHolder instanceof Group) {
+					getPerunBl().getAttributesManagerBl().setAttribute(sess, (Group) primaryHolder, attribute);
 			} else {
 				throw new InternalErrorException("Primary holder for attribute is not supported: " + primaryHolder);
 			}
```

Comment:
The first one is not needed, but since we ask about the `primaryHolder` earlier and throw `InternalErrorException` if it is unsupported class, the last condition will be true if visited. So I have removed the last else and the //noinspection labels.